docs/events.rst

```diff
@@ -19,18 +19,18 @@ Generally, you can listen to an event like this:
 
     # possibility 1
     @bot.event
-    async def on_(...):
+    async def on_xxx(...):
```

Comment:
Can you add this link https://discord.com/developers/docs/topics/gateway-events#receive-events

Reply:
this link is already present in this page line 154